### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by anandkesari**
Added a link to the top-level documentation for gRPC.

### CODE REVIEW

**Comment by anandkesari** [comment]
Signed the CLA.

**Comment by anandkesari** [comment]
I signed it.
On Jun 16, 2015 11:38, googlebot <notifications@github.com> wrote:Thanks for your pull request.  It looks like this may be your first contribution to a Google open source project, in which case you'll need to sign a Contributor License Agreement (CLA).

 Please visit https://cla.developers.google.com/ to sign.

Once you've signed, please reply here (e.g. I signed it!) and we'll verify.  Thanks.



If you've already signed a CLA, it's possible we don't have your GitHub username or you're using a different email address.  Check your existing CLA data and verify that your email is set on your git commits.If you signed the CLA as a corporation, please let us know the company's name.

File: src/cpp/README.md [hunk 1]
```diff
@@ -7,3 +7,5 @@ This directory contains source code for C++ implementation of gRPC.
 
 Alpha : Ready for early adopters
 
+#Reference
+[gRPC Documentation](https://github.com/grpc/grpc-common)
```
Comment (LisaFC) [inline src/cpp/README.md]: We're actually moving most of the user docs (other than installation instructions/release notes that live in the individual language repos) into our public site at grpc.io, can you change the link to http://www.grpc.io/docs/? Thanks!

**Comment by ctiller** [comment]
Is this dead?

**Comment by LisaFC** [comment]
This does seem to be dead, in that the edit I suggested was never implemented and the link as is in the fix doesn't exist anymore.

**Pull request closed without merging**